meta-ibm/recipes-phosphor/webui/webui-vue/changeLogo.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+model=($(busctl get-property xyz.openbmc_project.Inventory.Manager /xyz/openbmc_project/inventory/system/chassis/motherboard/bmc xyz.openbmc_project.Inventory.Decorator.Asset SubModel))
+echo ${model[1]}
+DIR=/usr/share/www/img
+if [ "${model}" = '"J0"' ]; then
```

Comment:
The request in github issue was to also handle the OEM scenario (show a blank logo if OEM brand)

Reply:
Done